app/models/reimbursement/expense/mileage.rb

```diff
@@ -34,7 +34,7 @@ module Reimbursement
   class Expense
     class Mileage < ::Reimbursement::Expense
       def rate
-        event.plan.mileage_rate(created_at)
+        event.plan.mileage_rate(created_at || Date.today)
```

Comment:
Hmm - what's the thought behind this?

Reply:
When creating a new mileage reimbursement using the dropdown button, the reimbursement record was not created yet, and `created_at` would be nil, causing an error in `mileage_rate`.

Reply:
Sounds good - I added a comment to explain this